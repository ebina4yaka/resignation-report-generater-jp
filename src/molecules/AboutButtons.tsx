import { ReactElement } from 'react'
import Grid from '@material-ui/core/Grid'
import Button from '@material-ui/core/Button'
import { createStyles, makeStyles } from '@material-ui/core/styles'
import Link from 'next/link'

const useStyles = makeStyles((theme) =>
  createStyles({
    root: {
      margin: theme.spacing(0, 0, 4),
    },
    button: {
      margin: theme.spacing(0, 2, 0),
      width: '20ch',
    },
  })
)

export default function AboutButtons(): ReactElement {
  const classes = useStyles()
  return (
    <div className={classes.root}>
      <Grid container justify="center">
        <Grid item>
          <Link href="/create" passHref>
            <Button
              className={classes.button}
              variant="contained"
              color="primary"
            >
              退職届を作成する
            </Button>
          </Link>
        </Grid>
      </Grid>
    </div>
  )
}
