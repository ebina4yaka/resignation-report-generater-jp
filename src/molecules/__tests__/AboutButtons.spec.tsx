import React from 'react'
import renderer from 'react-test-renderer'
import AboutButtons from '../AboutButtons'

test('Snapshot test for AboutButtons', () => {
  const tree = renderer.create(<AboutButtons />).toJSON()
  expect(tree).toMatchSnapshot()
})
