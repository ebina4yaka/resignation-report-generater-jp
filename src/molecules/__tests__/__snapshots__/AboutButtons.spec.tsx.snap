// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot test for AboutButtons 1`] = `
<div
  className="makeStyles-root-1"
>
  <div
    className="MuiGrid-root MuiGrid-container MuiGrid-justify-xs-center"
  >
    <div
      className="MuiGrid-root MuiGrid-item"
    >
      <a
        aria-disabled={false}
        className="MuiButtonBase-root MuiButton-root MuiButton-contained makeStyles-button-2 MuiButton-containedPrimary"
        href="/create"
        onBlur={[Function]}
        onClick={[Function]}
        onDragLeave={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onKeyUp={[Function]}
        onMouseDown={[Function]}
        onMouseEnter={[Function]}
        onMouseLeave={[Function]}
        onMouseUp={[Function]}
        onTouchEnd={[Function]}
        onTouchMove={[Function]}
        onTouchStart={[Function]}
        tabIndex={0}
      >
        <span
          className="MuiButton-label"
        >
          退職届を作成する
        </span>
      </a>
    </div>
  </div>
</div>
`;
